src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStepExecution.java

```diff
@@ -40,7 +46,9 @@
     @Override
     public boolean start() throws Exception {
 
-        Cloud cloud = Jenkins.getInstance().getCloud(step.getCloud());
+        LOGGER.info("starting");
```

Comment:
can you make this more descriptive and probably just `FINE` level ?

Reply:
I will remove this.